### DISCUSSION TITLE
[Solved] Can't start Supabase: Issue with Docker

### DISCUSSION THREAD

**Discussion by AugLuk** [mckaywrigley/chatbot-ui#1889] [Help]
**TL;DR: If you're getting public.ecr.aws related errors, restarting your PC might help**

Locally running Chatbot UI used to work for me in the past, but today it stopped working. I want to share how I solved it in case someone else runs into the issue.

When I ran `npm run chat`, it hung on the `starting databasebase...` step for a minute, then produced:
```
failed to pull docker image: Error response from daemon: Get "https://public.ecr.aws/v2/": dial tcp: lookup public.ecr.aws on 127.0.0.53:53: no such host
```

I've tried doing these but it didn't help:
- Restarting Docker (using Docker Desktop)
- Stopping all Supabase containers in Docker (using Docker Desktop)
- Upgrading Supabase CLI from 1.142.2 to 2.1.1
- Running `docker logout public.ecr.aws`
- Cause I recently upgraded Postgress, I tried removing `supabase/.temp/postgres-version` but there was no such file

What clued me in was running `docker ps -a` and seeing that some containers were missing that were visible in Docker Desktop. This was probably a bug with Docker, and all I needed to do was restart my PC for it to resolve.

**Comment by qfeng00** [comment]
hello， I failed installing supabase,could you please help me on this? Tapping supabase/tap
Cloning into '/opt/homebrew/Library/Taps/supabase/homebrew-tap'...
Username for 'https://gitee.com':
